bench_test.go

```diff
@@ -189,6 +198,30 @@ func BenchmarkReadRandomLmdb(b *testing.B) {
 	})
 }
 
+func BenchmarkReadRandomBolt(b *testing.B) {
+	boltdb := getBoltDB()
+	defer boltdb.Close()
+
+	runRandomReadBenchmark(b, "bolt", func(c *hitCounter, pb *testing.PB) {
+		err := boltdb.View(func(txn *bolt.Tx) error {
+			boltBkt := txn.Bucket([]byte("bench"))
+			y.AssertTrue(boltBkt != nil)
+			for pb.Next() {
+				key := newKey()
+				v := boltBkt.Get(key)
+				if v == nil {
+					c.notFound++
+					continue
+				}
+				y.AssertTruef(len(v) == *flagValueSize, "Assertion failed. value size is %d, expected %d", len(v), *flagValueSize)
```

Comment:
Avoid the `f` suffixes on Asserts unless you absolutely need them. They're much slower.